tcg/rarity.ts

```
@@ -1,41 +1,42 @@
-    ctx.restore();
-  }
-}
+import Canvas from 'canvas';

+import { DrawableFixed } from './interfaces/drawable';

+import { tcgAssetPaths } from './assetPaths';

+

+/**
```

ESLint

[error] More than 1 blank line not allowed.

(no-multiple-empty-lines)